src/components/number-picker/number-picker.component.less

```diff
@@ -4,7 +4,7 @@ ux-number-picker-inline {
     flex-direction: row;
     height: 34px;
 
-    &.ux-number-picker-invalid:not(.ng-invalid) {
+    &.ux-number-picker-invalid:not(.ng-valid) {
```

Comment:
Not sure about this change. If someone is using this property to manually control validity, they probably want that rule to take precedence over the form's validation state.